scanner/src/main/kotlin/scanners/fossid/FossIdConfig.kt

```diff
@@ -141,4 +146,20 @@ internal data class FossIdConfig(
 
         return FossIdNamingProvider(namingProjectPattern, namingScanPattern, namingConventionVariables)
     }
+
+    /**
+     * Create the [FossIdRestService] to interact with the FossID instance described by this configuration.
+     */
+    fun createService(): FossIdRestService {
+        val client = OkHttpClientHelper.buildClient()
+        log.info { "FossID server URL is $serverUrl." }
```

Comment:
Could you please also modify  `log4j2.xml` to add this class ?
Since you moved the log message to this class it won't get logged by default.